docs/md/content/usage/tool-authoring.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Authoring Tools"
+description: "Learn how to author a PRM tool."
+category: narrative
+tags:
+  - templates
+  - usage
+weight: 20
+---
+
+## Structure
+
+A PRM tool package can be packaged as a `.tar.gz` or hosted in a Github repository.
+It is comprised of a `prm-config.yml` and an optional `content` directory.
+The `content` directory can contain scripts / binaries / config files, etc... required to supplement the functionality of the tool outside the configurable parameters within the `prm-config.yml`.
```

Comment:
Add that the contents of this directory are placed in the /tmp directory. So 'content/myfile.sh' will be acessible at `/tmp/myfile.sh`

Reply:
I'll add something to cover that, although I am trying to strike a balance between being specific enough for the first iteration (Docker backend) and keeping it generic enough so a complete rewrite isn't required when we implement other backends.